nova/virt/vmwareapi/vim_util.py

```diff
@@ -25,6 +25,11 @@
 CONF = nova.conf.CONF
 
 
+class EmptyRetrieveResult:
+    def __init__(self):
+        self.objects = []
```

Comment:
Is this an instance instead of a class attribute, because you fear, that someone might add stuff to the list, which would then be kept forever and other receivers of `EmptyRetrieveResult` will also see it?

Reply:
yes. i mean for all the current use cases (~5 different places) it wouldn't be an issue, but import-time instantiation of a list just feels... icky.

Reply:
and it's only one more line of code, and a super familiar one at that.

Reply:
is this worth a mention in the commit-message or a code comment? people (upstream) might wonder.